tests/tests/unit_constants.rs

```diff
@@ -0,0 +1,25 @@
+use rune::Hash;
+
+#[test]
+fn test_get_const() -> rune::Result<()> {
+    let context = rune_modules::default_context()?;
+
+    let mut sources = rune::sources! {
+        entry => {
+        pub const LOAD_COUNT = 1337;
```

Comment:
Also want to test for these (since I'm not sure they work right now!):
* A constant exported in a nested public module.
* A constant re-exported (through `pub use ...` from a nested private module.

Reply:
Re-exports only work for functions but I'll fix that :D